packages/components/src/utils/input/input-control.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/core';
+
+/**
+ * Internal dependencies
+ */
+import { inputStyleNeutral, inputStyleFocus } from './base';
+import { font } from '../font';
+import { color } from '../colors';
+import { breakpoint } from '../breakpoint';
+
+export const inputControl = css`
+	font-family: ${ font( 'default.fontFamily' ) };
+	padding: 6px 8px;
+	${ inputStyleNeutral };
+
+	/* Fonts smaller than 16px causes mobile safari to zoom. */
+	font-size: ${ font( 'mobileTextMinFontSize' ) };
+	/* Override core line-height. To be reviewed. */
+	line-height: normal;
+
+	${ breakpoint( 'small' ) } {
```

Comment:
@saramarcondes Interesting ❤️! The [breakpoint mixin](https://github.com/ItsJonQ/g2/blob/master/packages/styles/src/mixins/breakpoints.js) I created for G2 uses a different API. I quite like the simplicity of this one for "inline" styling.

I think this technique works well for how you've been sprinkling in mixins within the CSS-in-JS files 👍